internal/api/server.go

```diff
@@ -0,0 +1,43 @@
+package api
```

Comment:
May I suggest we place it into `/test` or `/testing` or `/internal/test[ing]`? The `/internal/api` might look like it's not test-related.

See also: https://github.com/golang-standards/project-layout

Reply:
Maybe we can even place this file on the root level and make the whole test server stuff non-exportable

Reply:
Although internal cannot be exported, in order to simplify the project structure, it is better to put it in the root directory

Reply:
Another problem is that the package of the test file is different from that of the normal file. Therefore, if the method is set to not exportable, it cannot be accessed